MapboxNavigation/Abbreviations.swift

```diff
@@ -0,0 +1,129 @@
+import UIKit
+
+// http://wiki.openstreetmap.org/wiki/Name_finder:Abbreviations#English
+let Abbreviations = [
+    "apartments": "Apts",
```

Comment:
In order to make these strings localizable, let’s factor them out into a plist along the lines of Shields.plist. Unlike that file, I think Abbreviations.plist should consist of a dictionary with three keys – `Abbreviations`, `CompassDirections`, `Classifications` – each set to a dictionary mapping spelled-out words to their abbreviations. It’ll be a lot easier to localize a separate plist than strings hard-coded in a Swift file.